packages/ecommerce-app-base/src/SkuPicker/ProductSelectionList/ProductSelectionListItem.tsx

```diff
@@ -97,7 +100,7 @@ export const ProductSelectionListItem = (props: Props) => {
   const productIsMissing = !product.name;
 
   return (
-    <div className={styles.productWrapper}>
+    <div className={styles.productWrapper} data-testid="ProductSelectionListItem">
```

Comment:
What confuses me is that in some places we have a `data-testid` and in other - `data-test-id`. Why is that? Maybe worth changing that?

Reply:
`data-testid` is the default by testing library and `data-test-id` is the forma 36 override (https://github.com/contentful/forma-36/blob/2b6d93caaf3f870919bb60745634462e3456a615/scripts/test/setupTests.js).